apps/aehttp/priv/swagger.json

```diff
@@ -1623,6 +1687,42 @@
         }
       }
     },
+    "Contract" : {
+      "type" : "object",
+      "properties" : {
+        "code" : {
+          "type" : "string"
+        },
+        "options" : {
+          "type" : "string"
+        }
+      }
+    },
+    "ByteCode" : {
+      "type" : "object"
+    },
+    "ContractCall" : {
+      "type" : "object",
+      "properties" : {
+        "code" : {
```

Comment:
is this the hash to the contract byte code on the chain?

Reply:
No this is for off-chain call where "code" is the code encoded as a hex string.